testing/src/cypress/custom-command/institution/common-helper-functions.cy.ts

```diff
@@ -75,4 +80,15 @@ export default class InstitutionHelperActions {
   getUniqueId() {
     return v4();
   }
+
+  getRandomInstitutionCode() {
```

Comment:
Please add a TODO comment that this is a temporary code and it must be reviewed in upcoming PRs.

Reply:
Added a TODO mentioning the same